back-end/resources/room_config.json

```diff
@@ -70,19 +76,29 @@
         "turningWheel": "array"
       },
       "output": {
-        "audio": "string",
-        "ringTone": "string"
+        "audio": {
+          "type": "string",
```

Comment:
wouldn't status_type be more clear?

Reply:
This should be clear enough with the explanation in the documentation, otherwise, we can change this name to output_type or something along those lines.